container/matchers_test.go

```diff
@@ -0,0 +1,29 @@
+package container
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDockerImg(t *testing.T) {
+	assert := assert.New(t)
+
+	di := &DockerImg{
+		Repo: "some/image",
+		Tag:  "23",
+	}
+
+	assert.Equal("[Docker Image] Repo: "+di.Repo+", Tag: "+di.Tag, di.String())
+}
+
+func TestDockerContainer(t *testing.T) {
+	assert := assert.New(t)
+
+	dc := &DockerContainer{
+		ID:   "2432df023",
+		Name: "linus_foobar",
+	}
+
+	assert.Equal("[Docker Container] ID: "+dc.ID+", Name: "+dc.Name, dc.String())
```

Comment:
container package will need rework. This PR is mostly about the ```Pkg```. But yeah I agree. There are two new branches for both ```File``` and ```Service```